certloader/spiffe_tls_config.go

```diff
@@ -35,19 +35,19 @@ type spiffeLogger struct {
 }
 
 func (l spiffeLogger) Debugf(format string, args ...interface{}) {
-	l.log.Printf("(spiffe) [DEBUG]: "+format, args...)
+	l.log.Printf("spiffe/debug: "+format, args...)
```

Comment:
Is that an established ghostunnel logging format? I'm asking because my initial reaction would be to read this as `component/subcomponent` logger.

Reply:
Not an established format -- the SPIFFE logger is the only thing that has debug/warn/error/etc levels, everything else just uses the built-in Go logger. Could change this in a future version